standalone/src/models/order/payment-info/builders.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { Payment } from '@/models/payment';
+import { PaymentInfoRest, PaymentInfoGraphql } from './index';
+
+describe('PaymentInfo Builder', () => {
+  it('should build properties for the REST representation', () => {
+    const restModel = PaymentInfoRest.random().build();
+
+    expect(restModel).toEqual(
+      expect.objectContaining({
+        payments: expect.any(Array),
+      })
+    );
+  });
+  it('should build properties for the GraphQL representation', () => {
+    const graphqlModel = PaymentInfoGraphql.random().build();
+
+    expect(graphqlModel).toEqual(
+      expect.objectContaining({
+        payments: expect.any(Array),
```

Comment:
We must expect the items in the `payments` should have the `__typename` value equals to `Payment`.
Also, items in the `paymentRefs` should be objects with a `__typename` of `Reference` and a `typeId` of `payment`.

Reply:
Fixed here: 643bfea2740f49cc2a2506e74deec94e285cf109